tldw_Server_API/app/api/v1/endpoints/admin/admin_usage.py

```
@@ -269,3 +269,16 @@ async def export_llm_usage_csv(
     resp = PlainTextResponse(content=content, media_type="text/csv")
     resp.headers["Content-Disposition"] = 'attachment; filename="llm_usage_export.csv"'
     return resp
+
+
+@router.get("/usage/cost-attribution")
+async def get_cost_attribution(
+    group_by: str = Query("user", description="Group by: user or org"),
+    range_days: int = Query(7, ge=1, le=90),
+    principal: AuthPrincipal = Depends(get_auth_principal),
```

Ruff (0.15.7)

[warning] Unused function argument: `principal`

(ARG001)